src/style.css

```diff
@@ -0,0 +1,72 @@
+*,
+::before,
+::after {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+main {
+  margin: 10% 15% 10% 50%;
+  transform: translateX(-50%);
+  border: 1px solid rgb(245, 245, 245);
+  box-shadow: 3px 2px 10px grey;
+}
+
+main header h3 {
+  border-bottom: 1px solid rgb(221, 220, 220);
+  padding: 1rem 0;
+  padding-left: 0.8rem;
+  margin-bottom: 0;
+  margin-top: 0;
+}
+
+label > i,
+main header h3 > i {
+  float: right;
+  margin-right: 2rem;
+}
+
+.add-task {
+  display: flex;
+  align-items: baseline;
+  border-bottom: 1px solid rgb(221, 220, 220);
+}
+
+.icon {
+  margin-right: 2rem;
+}
+
+.add-task input {
+  outline: none;
+  padding: 1rem 0;
+  padding-left: 0.8rem;
+  border: none;
+  width: 97%;
+}
+
+ul {
+  margin-top: 0;
+  padding-left: 0;
+}
+
+li {
+  list-style-type: none;
+  border-bottom: 1px solid rgb(221, 220, 220);
+  padding: 1.3rem 0;
+  padding-left: 0.8rem;
+}
+
+[type~="checkbox"] {
+  cursor: pointer;
+}
+
+label {
+  margin-left: 0.8rem;
+}
+
+.clear-task h4 {
+  padding: 1.5rem 0.7rem;
+  text-align: center;
+  color: grey;
+}
```

Comment:
Let's improve on the UI design (font, color, spacing, icon) so it looks similar to the required design. 👍 

> Your design
> ![Your design](https://user-images.githubusercontent.com/66112760/144213494-76325ab5-d7ab-4f43-a217-1f14e03646f8.png)  

> Required design
> ![Screenshot (328)](https://user-images.githubusercontent.com/66112760/144213738-c5493a65-d627-49d3-8618-aea0d70349c1.png)